optimum/neuron/utils/runner.py

```diff
@@ -213,7 +214,7 @@ def __init__(
         self.use_venv = use_venv
         self.should_install_requirements = install_requirements
         self.venv_dir = TemporaryDirectory()
-        self.python_name = "python"
+        self.python_name = sys.executable
         self.pip_name = "pip"
         self.torchrun_name = "torchrun"
```

Comment:
This case still works considering that you needed to change the name for `self.python_name` and we either use `self.python_name` or `self.torchrun_name`?